### ISSUE TITLE
GM_download.abort() does not function

### ISSUE THREAD

**Issue by L0laapk3** [Tampermonkey/tampermonkey#494]
`GM_download.abort()` Does not stop the download.
Tested on v4.5 and v4.5.5663.
Consider the following example:
```
// ==UserScript==
// @name         test script
// @match        https://github.com/
// @grant        GM_download
// ==/UserScript==

var dlObject = GM_download({
    url: "http://ipv4.download.thinkbroadband.com/5MB.zip",
    name: "test.zip"
});
dlObject.abort();
```

**Comment by L0laapk3** [comment]
Can somebody confirm or deny that this is indeed a problem with tampermonkey and not on my end?

**Comment by derjanb** [comment]
Yes, it's a bug and it will be fixed at the next beta version.

**Comment by derjanb** [comment]
Fixed at TM 4.6.5694
